packages/did/package.json

```diff
@@ -42,6 +42,7 @@
     "@polkadot/api": "^9.0.0",
     "@polkadot/keyring": "^10.0.0",
     "@polkadot/types": "^9.0.0",
+    "@polkadot/types-codec": "9.1.1",
```

Comment:
why would we pin this?

Reply:
Done in https://github.com/KILTprotocol/sdk-js/commit/e1edbc3daa11062c3d173e5ba7b76f84cc14c945